### DISCUSSION TITLE
Login to Windows using Ctrl, Alt & Del on a Mac Keyboard

### DISCUSSION THREAD

**Discussion by TicT0ck** [utmapp/UTM#4454] [UTM Help (macOS)] [answered]
I have a MacBook Air M1.  I have Server 2022 installed in UTM Virtual Machine.  I have successfully installed Server 2022 and it is asking me to press CTRL, ALT & DEL but how do I this from a MacBook Air laptop?

**Comment by conath** [comment, marked as answer]
On Mac you can get the DEL key by pressing fn and delete together.
So you want to press  fn ⌃ ⌥ delete

**Comment by BurhanV** [comment]
Even I did install 2022 on UTM somehow. IT is huge slow, but getting managed somehow. I was stuck on the same thing and was searching for it. Got me here.

**Comment by kg954** [comment]
I had the same problem. I tried fn+control+option+delete, it worked
